src/test/java/org/elasticsearch/search/aggregations/bucket/TopHitsTests.java

```diff
@@ -531,37 +529,6 @@ public void testFailWithSubAgg() throws Exception {
             assertThat(e.getMessage(), containsString("Aggregator [top_tags_hits] of type [top_hits] cannot accept sub-aggregations"));
         }
     }
-    
-    @Test
```

Comment:
As we have changed the functionality, should we now have a test which explicitly makes sure the aggregation is run when you run top hits on a breath_first terms agg? i.e. confirm that the setting is indeed ignored. Or is this already done elsewhere?